views/update/overseas-entity-query.html

```diff
@@ -0,0 +1,35 @@
+{% extends "layout.html" %}
+
+{% set title = "What is the OE number? (NOT LIVE)" %}
+
+{% block pageTitle %}
+{% include "includes/page-title.html" %}
+{% endblock %}
+
+{% block backLink %}
+{% include "includes/back-link.html" %}
+{% endblock %}
+
+{% block content %}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+        {% include "includes/list/errors.html" %}
+        <h1 class="govuk-heading-xl">What is the OE number? (NOT LIVE)</h1>
```

Comment:
Same here, we need the `NOT LIVE`

Reply:
same as @AndreKramer7654 has said above

Reply:
https://github.com/companieshouse/overseas-entities-web/pull/481 has been merged shortly after